src/features/world/scenes/BaseScene.ts

```diff
@@ -265,12 +283,7 @@ export class BaseScene extends Phaser.Scene {
             this.cameras.main.on(
               "camerafadeoutcomplete",
               () => {
-                console.log("Fade complete");
-                this.roomService.send("CHANGE_ROOM", {
-                  roomId: warpTo,
-                });
-
-                this.game.scene.switch(this.scene.key, warpTo);
+                this.scene.start(warpTo);
```

Comment:
`this.scene.start` shuts down the previous scene, and starts a new scene. I think previously the plaza was hidden underneath the auction house, and was still receiving events. I believe this change stops both scenes rendering at the same time.